From 920f2a3beeca4564b31671ea86bd6b7cde716c5c Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@st.com>
Date: Fri, 21 Oct 2022 11:41:38 +0200
Subject: [PATCH 049/954] feat(stm32mp15-fdts): adapt stm32mp157a-dk1 board to
 stm32 DT diversity

To handle STM32MP15 SOCs diversity, some updates have to been done.
This commit mainly adapts stm32mp157a-dk1 board to include the
correct SOC version.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git d67f86cc39e126454cd9b11ff482338804253dcd feat(stm32mp15-fdts): adapt stm32mp157a-dk1 board to stm32 DT diversity]
Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I538582b867bbbbf493e4fd5cb36ef68125880349
---
 fdts/stm32mp157a-dk1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fdts/stm32mp157a-dk1.dts b/fdts/stm32mp157a-dk1.dts
index b4d5d20cd1..bb0ffcf5f0 100644
--- a/fdts/stm32mp157a-dk1.dts
+++ b/fdts/stm32mp157a-dk1.dts
@@ -7,6 +7,7 @@
 /dts-v1/;
 
 #include "stm32mp157.dtsi"
+#include "stm32mp15xa.dtsi"
 #include "stm32mp15-pinctrl.dtsi"
 #include "stm32mp15xxac-pinctrl.dtsi"
 #include "stm32mp15xx-dkx.dtsi"
-- 
2.53.0

